HPC/Gromacs.md

```diff
@@ -0,0 +1,98 @@
+## Gromacs
+
+Gromacs is one of the most widely used molecular dynamics software packages. One important way for Gromacs to achieve high performance is to use Modern processors' SIMD (single instruction multiple data) capabilities to increase floating-point throughput. An SVE implementation for Gromacs inner kernel has been added to the latest release, Gromacs 2022.4. SVE implementation gives 10% better performance than the previous NEON SIMD implemntation on Hpc7g/c7gn instances which support both SVE and NEON vectorization. Below is the recommended SIMD option for different aarch64 architectures:
+* Hpc7g/c7gn (-DGMX_SIMD=ARM_SVE)
+* c6gn (-DGMX_SIMD=ARM_NEON_ASIMD)
+
+For FFT, you can use both ArmPL FFT and FFTW library. ArmPL FFT has better performance, but the impact of different FFT libraries to the following 3 Gromacs benchmarks performance is small. Gromacs supports 3 different programming interfaces for FFT: "fftw3", "mkl" and "fftpack". ArmPL FFT library has the same programming interface as FFTW and setting "-DGMX_FFT_LIBRARY=fftw3" and "-DFFTWF_LIBRARY=${ARMPL_LIBRARIES}/libarmpl_lp64.so" allows ArmPL FFT library for Gromacs. Setting "-DGMX_FFT_LIBRARY=fftw3" and "-DFFTWF_LIBRARY=/shared/fftw-3.3.9/lib/libfftw3f.a" will link Gromacs with FFTW library.
```

Comment:
'For FFT, you can use both, the ArmPL FFT and the FFTW library'
'The ArmPL FFT library has the same programming'
'lp64.so" enables the ArmPL FFT library for Gromacs'
'will link Gromacs with the FFTW library'